app/controllers/admin/booths_controller.rb

```diff
@@ -50,6 +50,7 @@ def create
 
     def edit
       @url = admin_conference_booth_path(@conference.short_title, @booth.id)
+      @invitation_pending_emails = @booth.responsibles.where(last_sign_in_at: nil).pluck(:email).join(',')
```

Comment:
Should we extract this to a booth method `invited_responsibles` or something like that?

Reply:
Yes, we can do that.